bundles/org.openhab.binding.prowl/src/main/resources/OH-INF/i18n/prowl_fr.properties

```diff
@@ -30,5 +30,3 @@ pushNotificationActionEventLabel = Evénement
 pushNotificationActionEventDescription = Nom de l'événement.
 pushNotificationActionMessageLabel = Message
 pushNotificationActionMessageDescription = Texte du message.
-pushNotificationActionPriorityLabel = Priorit�
-pushNotificationActionPriorityDescription = Priorit� du message (-2,-1,0,1,2).
```

Comment:
This should not be removed.
The problem was introduced by PR #12647 which added these two french translations.

Reply:
I am going to upload the file in Crowdin.

Reply:
Unfortunately, even after uploading the right file in Crowdin, it does not revert then unexpected change in the PR.
I guess the PR has to be rebased but I don't feel comfortable with this action. @wborn: can you please do it ?

Reply:
In fact, I have the feeling that the real problem is that the English file prowl.properties was not synchronized with Crowdin and so the 2 new entries are unknown in Crowdin.

Reply:
I have uploaded manually the English string in Crowdin. I am going to close this PR and create a new one.